ocaml/xapi/xapi_globs.ml

```diff
@@ -1594,6 +1598,22 @@ let other_options =
     , (fun () -> string_of_bool !disable_webserver)
     , "Disable the host webserver"
     )
+  ; ( "enable-external-authentication-cache"
+    , Arg.Set external_authentication_cache_enabled
+    , (fun () -> string_of_bool !external_authentication_cache_enabled)
+    , "Enable caching of external authentication decisions"
+    )
+  ; ( "external-authentication-expiry-ms"
+    , Arg.Int (fun ms -> external_authentication_expiry_ms := ms)
```

Comment:
`options_of_xapi_globs_spec` has logic to transform from floats (and ints) in seconds to Mtime.Span.t.
It doesn't currently allow to set a custom description and instead it generates it. It uses the variant ShortDurationFromSeconds

It shouldn't be difficult to decouple the generation of the description from the rest of the transformations, allowing to have two lists: the current one (`xapi_globs_spec`), without descriptions, and a new one, with descriptions, where you can place these.

Reply:
Yes, that's just what I wanted. Thanks!